.changeset/dry-flowers-shave.md

```diff
@@ -0,0 +1,6 @@
+---
+"@latticexyz/common": patch
+"@latticexyz/react": patch
+---
+
+Removed some unused files, namely `curry` in `@latticexyz/common` and `useDeprecatedComputedValue` from `@latticexyz/react`.
```

Comment:
for future reference, you can create a changeset for each package with the package-specific changes so that we don't need the irrelevant change in another package's changelog